src/app/components/add-word/add-word.component.html

```diff
@@ -13,23 +13,26 @@ <h1>
     </div>
   </h1>
   
-  <form name="formAddWord" action="" #form= "ngForm" novalidate>
+  <form name="formAddWord" action="" #form= "ngForm" novalidate [formGroup]="userForm">
     <div class="form-group">
-      <input type="text" name="word" class="form-control" [(ngModel)]="word.word" required />
+      <input type="text" formControlName="word"  name="word" class="form-control" id="word" [(ngModel)]="word.word"  required />
```

Comment:
we are not using ID property any where, remove it and use some other selector

Reply:
fixed